Profiles/mp-MedicationAdministration2.xml

```diff
@@ -388,11 +388,10 @@
     </element>
     <element id="MedicationAdministration.performer.actor">
       <path value="MedicationAdministration.performer.actor" />
-      <short value="Patient / HealthcareProvider / Caregiver" />
-      <definition value="(The healthcare provider of) the person who realized the administration. It refers to either the organization for which the professional responsible for administration works, the patient himself or, for example, the informal carer." />
-      <comment value="Whenever a HealthcareProvider is responsible for the medication administration, this concept refers to a PractitionerRole instance in which the nl-core-HealthcareProvider-Organization instance is referred to in `PractitionerRole.organization`. This use of the PractitionerRole resource type is described in the current FHIR build as: 'For use cases where an organization has activities where a practitioner is not defined/pre-allocated for a specific role (e.g. an un-named surgeon at XYZ Hospital), a PractitionerRole resource can be used with an empty Practitioner property, and the other relevant role properties populated'." />
+      <short value="Patient / HealthProfessional / Caregiver" />
+      <definition value="(The health professional of) the person who realized the administration. It refers to either the health professional responsible for the administration , the patient himself or, for example, the informal carer." />
```

Comment:
This definition is not entirely correct. I'd change this to the definition present before the changes in https://github.com/Nictiz/Nictiz-R4-MedicationProcess/pull/19 were done.